dask/dataframe/tests/test_rolling.py

```diff
@@ -574,6 +575,7 @@ def test_groupby_rolling():
     assert_eq(expected, actual, check_divisions=False)
 
 
+@pytest.mark.xfail(dd._dask_expr_enabled(), reason="this works in dask-expr")
```

Comment:
Not sure I understand the `reason` here?

Reply:
Dask raises but we made this work in dask-expr, the tests checks for a ValueError, but dask-expr just completes without raising, so the test will fail